buildspec.yaml

```diff
@@ -0,0 +1,19 @@
+version: 0.2
```

Comment:
Specify a yaml version 1.1 declaration (SAM supports YAML 1.1 currently and I'd like a way to ensure that we keep things up to date)